components/CategoriesList.js

```diff
@@ -1,10 +1,13 @@
 import styled from "styled-components";
-import { useState } from "react";
+import { useEffect, useState } from "react";
```

Comment:
```suggestion
import { useEffect, useState } from "react";
```
Are you actually using `useEffect` anywhere?